frontend/src/app/supportAdmin/PendingOrganizations/modals/VerifyPendingOrganizationConfirmationModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import Modal from "react-modal";
+
+import Button from "../../../commonComponents/Button/Button";
+import { PendingOrganization } from "../../../../generated/graphql";
+
+interface VerifyConfirmationModalProps {
+  onVerifyConfirm: () => void;
+  onClose: () => void;
+  setVerifyConfirmation: (verify: boolean) => void;
+  organization: PendingOrganization;
+}
+
+const VerifyPendingOrganizationConfirmationModal: React.FC<
+  VerifyConfirmationModalProps
+> = ({ onVerifyConfirm, onClose, setVerifyConfirmation, organization }) => {
+  return (
+    <Modal
```

Comment:
⚠️ This file is refactoring this `<Modal>` into its own file: https://github.com/CDCgov/prime-simplereport/blob/0afcc71b7c3998d23ddf7ee1ddee00e7978973dd/frontend/src/app/supportAdmin/PendingOrganizations/modals/ConfirmOrgVerificationModal.tsx#L104

Reply:
Not asking for a code change I just wanted to make you aware that there is a Modal implementation in the common components that saves some duplicated code. I used it in [here](https://github.com/CDCgov/prime-simplereport/blob/cb9ec6a8c7fdf293ccd45b638ca517d833eb74a5/frontend/src/app/supportAdmin/ManageFacility/FacilityInformation.tsx#L48). Using this one instead of the react-modal library directly saves some code IMO.